src/components/Cards/Cards.component.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const Card = styled.div`
+    display: flex;
+    flex-direction: column;
+    background-color: #191919;
+    color: #aaaaaa;
+    margin-right: 1.5rem;
+    margin-bottom: 2rem;
+    font-size: 1.4rem;
+`;
+
+const Image = styled.img`
+    width: 100%;
+    height: 15rem;
+    object-fit: cover;
+`;
+
+const Title = styled.h3`
+    color: white;
+    padding-bottom: 0.4rem;
+`;
+
+const InfoContainer = styled.div`
+    padding: 1rem;
+`;
+
+const Subtitle = styled.h4`
+    font-weight: normal;
+`;
+
+const Description = styled.p`
+    margin-top: 0.8rem;
+`;
```

Comment:
Same comment here. I would create a file called `Cards.styled.js` and there I would export all the styles and use them here